examples/complete/main.tf

```diff
@@ -37,3 +37,30 @@ module "create_scc_instance" {
   en_instance_crn                   = module.event_notification.crn
   skip_cos_iam_authorization_policy = false
 }
+
+locals {
+  scope = [{
+    environment = "ibm-cloud"
+    properties = [
+      {
+        name  = "scope_type"
+        value = "account.resource_group"
+      },
+      {
+        name  = "scope_id"
+        value = module.resource_group.resource_group_id
+      }
+    ]
+  }]
+}
+
+module "create_profile_attachment" {
+  source                 = "../../modules/attachments/"
+  profile_id             = "f54b4962-06c6-46bb-bb04-396d9fa9bd60" # defaulted to SOC 2
+  scc_instance_id        = module.create_scc_instance.guid
+  attachment_name        = "${var.prefix}-attachment"
+  attachment_description = "profile-attachment-description"
+  attachment_schedule    = "every_7_days"
+  attachment_status      = "enabled"
+  scope                  = var.scope != null ? var.scope : local.scope
+}
```

Comment:
please update the contents of `examples/complete/README.md` to include details of the kind of attachment the example is now creating.